exp-player/addon/components/exp-player.js

```diff
@@ -148,6 +148,9 @@ able to continue the study.
                 this._transition();
                 this.set('frameIndex', frameIndex + 1);
                 this.set('framePage', 0);
+                this.get('session').set('frameIndex', frameIndex + 1);
```

Comment:
What is the advantage of tracking the same property twice? For example, are there specific design barriers to directly binding the value from the session to the controller? (`Ember.computed.alias` or `Ember.computed.or`, etc)

Two way bindings can be weird sometimes, but this implementation is pretty explicit that whenever the internal state variable changes, the session is also modified and saved. In that case, a two way data binding might be ok and not a major design change.